cmd/helm/history_test.go

```diff
@@ -34,50 +35,31 @@ func TestHistoryCmd(t *testing.T) {
 		})
 	}
 
-	tests := []struct {
-		cmds string
-		desc string
-		args []string
-		resp []*rpb.Release
-		xout string
-	}{
+	tests := []releaseCase{
 		{
-			cmds: "helm history RELEASE_NAME",
-			desc: "get history for release",
+			name: "helm history RELEASE_NAME -- get history for release",
```

Comment:
I think on these names, let's just shorten to something like `get history for release` and drop the rest

Reply:
Thank you for the review. I have updated the PR using shortened names for the tests.